backend/routes/routes.go

```diff
@@ -1,11 +1,17 @@
 package routes
 
-func InitRoutes() {
-	InitIndexerRoutes()
-	InitCanvasRoutes()
-	InitPixelRoutes()
-	InitWebsocketRoutes()
-	InitTemplateRoutes()
-	InitUserRoutes()
-	InitContractRoutes()
+import (
+  "github.com/gorilla/mux"
+)
+
+func InitRoutes(router *mux.Router) { // THis will accepts the router instance as an argument
+  InitIndexerRoutes(router)
```

Comment:
I think you'll need to change all these route functions if we decide to swap to `gorilla/mux`